tests/integration_tests/job_queue/test_lsf_driver.py

```diff
@@ -176,11 +176,7 @@ def copy_lsf_poly_case(copy_poly_case, tmp_path):
     "mock_start_server",
 )
 @pytest.mark.integration_test
-def test_run_mocked_lsf_queue(request, using_scheduler):
-    if "fail" in request.node.name and using_scheduler:
-        pytest.skip(
-            "Python LSF driver does not support general resubmission on bsub errors"
-        )
```

Comment:
Does this work now?

Reply:
depends on what you mean by "work". I turned the test of for scheduler :sweat_smile:.

Reply:
Ahhhh, so that's what it did.
Should we not run those tests with scheduler?

Reply:
we have other tests covering the same code